core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -73,6 +73,28 @@ coreos:
 
         [Install]
         RequiredBy=format-etcd2-volume.service
+{{if .AmazonSsmAgent.Enabled}}
+    - name: amazon-ssm-agent.service
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=amazon-ssm-agent
+        Requires=network-online.target
+        After=network-online.target
+
+        [Service]
+        Type=simple
+        WorkingDirectory=/opt/bin/ssm
```

Comment:
Does running `aws ssm send-command` result in running commands in this working directory?
Any reason we won't make it `WorkingDirectory=/`?